include/cantera/oneD/StFlow.h

```diff
@@ -106,6 +106,15 @@ class StFlow : public Domain1D
         return m_do_soret;
     }
 
+    //! Compute species diffusive fluxes with respect to
+    //! their mass fraction gradients (fluxGradientBasis = ThermoBasis::mass)
+    //! or mole fraction gradients (fluxGradientBasis = ThermoBasis::molar, default)
+    //! when using the mixture-averaged transport model
+    void fluxGradientBasis(ThermoBasis fluxGradientBasis);
+    ThermoBasis fluxGradientBasis() const {
+        return m_fluxGradientBasis;
+    }
```

Comment:
Each of these functions needs a docstring so it will be handled correctly by Doxygen. You can use `@copydoc` or `@see` to avoid having to repeat the content.

For consistency, the setter should be `setFluxGradientBasis` rather than overloading the same name as the getter.